src/hammersbald.rs

```diff
@@ -179,12 +179,6 @@ impl ChainDB for Hammersbald {
     fn fetch_header(&self, id: &sha256d::Hash) -> Result<Option<StoredHeader>, Error> {
         Ok(self.db.get_hash_keyed::<StoredHeader>(id)?.map(|(_, header)| header))
     }
-
-    /// Shutdown db
-    fn shutdown(&mut self) {
```

Comment:
Shouldn't there be a `shutdown` method somewhere to stop the db? I think it should be moved into Hamersbald structure methods instead of completely removing it. That might also break some existing API, but those changes are warranted anyway.

Reply:
On every batch() call the hammersbald data will be flushed to disk and should be in a stable state, this is essentially all the shutdown() function does. I exposed it because I thought it fixed an issue I was having on Android, but I was wrong. I only recent added it to the API so I'm pretty sure no one else is using it.

Reply:
Oh my bad. I thought it was there already. 
ACK `f8902d`